python-sdk/src/forta_agent/utils.py

```diff
@@ -2,6 +2,9 @@
 import os
 from jsonc_parser.parser import JsoncParser
 import sha3
+import requests
+
+from forta_agent.forta_graphql import AlertsResponse
```

Comment:
we should use a relative import here i.e. `from .forta_graphql import AlertsResponse`

Reply:
Updated